vbml/validator/map.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, add_defaults: bool = True):
             for validator in default_validators:
                 self.add(validator)
 
-    def add(self, validator: ABCValidator) -> typing.NoReturn:
+    def add(self, validator: ABCValidator):
```

Comment:
correct type here is `--> None`, also to complete typing enhancement for this code you may add typing for `.get`

Reply:
The absence of a result annotation also means `None`. But if you put the strictly annotation, then it might be worth also returning `None` strictly.

Reply:
None result typing is more explicit for typecheckers. Return statement is redundant
Also .get still needs typing